Sources/Card/Models/Address.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+
+/// The portable international postal address.
+/// Maps to [AddressValidationMetadata](https://github.com/googlei18n/libaddressinput/wiki/AddressValidationMetadata)
+/// and HTML 5.1 [Autofilling form controls: the autocomplete attribute](https://www.w3.org/TR/html51/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute).
+public struct Address: Codable {
+
+    enum CodingKeys: String, CodingKey {
+        case addressLine1
+        case addressLine2
+        case locality = "admin_area_2"
+        case region = "admin_area_1"
+        case postalCode
+        case countryCode
+    }
+
+    /// The first line of the address. For example, number or street. For example, `173 Drury Lane`.
+    /// Required for data entry and compliance and risk checks. Must contain the full address.
```

Comment:
Looks like we use "for example" twice. Maybe this can be `The first line of the address, number and/or street. For example, 173 Drury Lane.` Similar for the second line as above, do merchants need to know why this is required or can that me removed? If it _is_ required maybe we can update it to read as "required when using PayPal's fraud tools" or whatever the merchant facing term is for when we run those checks.

Reply:
yea I think we won't need that, I'll shorten this to just be `The first line of the address`